test/ng/rootScopeSpec.js

```diff
@@ -212,10 +212,52 @@ describe('Scope', function() {
       expect($rootScope.$$watchersCount).toBe(2);
     }));
 
-    it('should not keep constant literals on the watch queue', inject(function($rootScope) {
+    it('should remove $watch of constant literals after initial digest', inject(function($rootScope) {
```

Comment:
I don't understand, isn't it the same as before and the same as all the others around it?